### PR TITLE
794 playlist smart ordering api

### PR DESCRIPTION
**Pull request by froveda**
Trello card: https://trello.com/c/ROr6JZSL/794-as-a-customer-developer-id-like-to-modify-playlist-smart-ordering-via-api-so-i-can-order-playlist-content-outside-of-admin-gui

### CODE REVIEW

File: api_docs/playlists.md [hunk 1]
```diff
@@ -185,3 +185,105 @@ video_id[]  | A comma separated list of video IDs to remove from the playlist |
   }
 }
 </pre>
+
+## Smart Ordering
+
+<p>Playlist smart ordering allow you to change the order of its videos by a sort options. The videos added or updated into a playlist with sort options, are sort automatically following that sort option</p>
```
Comment (trevordjones) [inline api_docs/playlists.md]: "Playlist smart ordering allows you" (allows)
"its videos by sort options" (remove 'a')
"The videos added or updated into a playlist with sort options are sorted automatically" (remove comma, sorted)

File: api_docs/playlists.md [hunk 2]
```diff
@@ -185,3 +185,105 @@ video_id[]  | A comma separated list of video IDs to remove from the playlist |
...
+
+Parameter | Function | Type
+--------- | -------- | ----
+playlist[sort_options] | An array of sort options. Only two levels of sorting are accepted, and the videos are sort first by the first level, and second as the second level | Array
+
```
Comment (trevordjones) [inline api_docs/playlists.md]: "are sorted first by first" (sorted)

File: api_docs/playlists.md (hunk 1 above)
Comment (froveda) [inline api_docs/playlists.md]: @trevordjones changed!

File: api_docs/playlists.md (hunk 2 above)
Comment (froveda) [inline api_docs/playlists.md]: @trevordjones changed!

**Pull request merged**